apps/api/src/country/country.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common'
+import { Public } from 'nest-keycloak-connect'
+import { CountryService } from './country.service'
+import { CreateCountryDto } from './dto/create-country.dto'
+import { UpdateCountryDto } from './dto/update-country.dto'
+
+@Controller('country')
+export class CountryController {
+  constructor(private readonly countryService: CountryService) {}
+
+  @Post('create-country')
+  @Public()
```

Comment:
This controller should not be `@Public()` for any action except for `@Get()` requests.